certified-connectors/Text Request/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3184 @@
+{
+  "swagger": "1.0",
```

Comment:
@connorjustbecause one of the checks is failing because of this line. The version for swagger should be `2.0`